Example/Example/CharactersViewController.swift

```diff
@@ -24,4 +24,13 @@ class CharactersViewController: ExampleViewController, BothamTableViewController
         pullToRefreshHandler.addTo(tableView)
         super.viewDidLoad()
     }
+    
+    override func viewDidAppear(animated: Bool) {
+        super.viewDidAppear(animated)
+        pullToRefreshHandler.beginRefreshing()
+        let delayTime = dispatch_time(DISPATCH_TIME_NOW, Int64(3 * Double(NSEC_PER_SEC)))
+        dispatch_after(delayTime, dispatch_get_main_queue()) {
+            self.pullToRefreshHandler.beginRefreshing()
+        }
```

Comment:
If this is just to test that the improvments on the BothamPullToRefreshHandler, I prefer that you write a test.